core/src/main/java/org/apache/carbondata/core/util/SessionParams.java

```diff
@@ -218,15 +217,7 @@ private boolean validateKeyValue(String key, String value) throws InvalidConfigu
         } else if (key.startsWith(CarbonCommonConstantsInternal.QUERY_ON_PRE_AGG_STREAMING)) {
           isValid = true;
         } else if (key.startsWith(CarbonCommonConstants.CARBON_DATAMAP_VISIBLE)) {
-          String[] keyArray = key.split("\\.");
-          isValid = DataMapStoreManager.getInstance().isDataMapExist(
-              keyArray[keyArray.length - 3],
-              keyArray[keyArray.length - 2],
-              keyArray[keyArray.length - 1]);
-          if (!isValid) {
-            throw new InvalidConfigurationException(
-                String.format("Invalid configuration of %s, datamap does not exist", key));
-          }
+          isValid = true;
```

Comment:
Why this validation is not needed? no need to check `isDataMapExist `?

Reply:
This check has been moved to org/apache/spark/sql/hive/execution/command/CarbonHiveCommands.scala:110